app/piiremover.py

```diff
@@ -3,25 +3,40 @@
 ni = compile('[a-zA-Z]{2}(?:\s*\d\s*){6}[a-zA-Z]?')
 phone = compile('(((\+44\s?\d{4}|\(?0\d{4}\)?)\s?\d{3}\s?\d{3})|((\+44\s?\d{3}|\(?0\d{3}\)?)\s?\d{3}\s?\d{4})|((\+44\s?\d{2}|\(?0\d{2}\)?)\s?\d{4}\s?\d{4}))(\s?\#(\d{4}|\d{3}))?')
 vrp = compile('((?:[a-zA-Z]{2}\s?[0-9]{2}\s?[a-zA-Z]{3})|(?:[a-zA-Z]{3}\s?\d{4}))')
-credit_cards = compile('(?:\d[ -]*?){13,16}')
```

Comment:
FYI pre-compiling regexes is no longer best practice in python, now that python caches the compiled regex the first time it is used. So you could save a tiny bit of start-up time by not compiling, but no biggie.